src/main/java/org/fundacion/jala/converter/view/converter/ResolutionVideo.java

```diff
@@ -0,0 +1,42 @@
+package org.fundacion.jala.converter.view.converter;
+
+public class ResolutionVideo {
+    private int width;
+    private int height;
+    private String name;
+
+    ResolutionVideo(String name,int width,int height ){
+        this.height=height;
+        this.width=width;
+        this.name=name;
+    }
+
```

Comment:
I think that is missing javadoc for the methods

Reply:
ok